resources/views/dinnerform/admin_includes/dinnerform-details.blade.php

```diff
@@ -0,0 +1,94 @@
+<form method="post"
```

Comment:
For consistency, maybe this file should be called "-edit"?

Reply:
This name is consistent with the naming of the event partials.

Reply:
should I still rename this partial, or should I keep it consistent with the events partials?

Reply:
Neh, it's okay like this.